js/components/utils.js

```diff
@@ -418,14 +418,20 @@ var Utils = {
   * @returns {String} The countdown time string
   */
   getStartCountdown: function(countdownTimestamp) {
+    var dayString = "day";
+    var hourString = "hour";
+    var minuteString = "minute";
     try {
       if (countdownTimestamp < 0) return "";
       var days = Math.floor(countdownTimestamp / (24 * 60 * 60 * 1000));
```

Comment:
for code maintainability, it would be better if you store the divisor day component in a variable and reuse it wherever you need it. Like 
```
var dayDivisor =  24 * 60 * 60 * 1000;
var days = Math.floor(countdownTimestamp / dayDivisor);
countdownTimestamp -= days * dayDivisor;
```

Same applies for the hours and minutes if necessary.